### PR TITLE
lists in formula

### PR DESCRIPTION
**Pull request by xithan**
support wolfram list syntax in ruby formulas

### CODE REVIEW

File: mod/calculated_metrics/lib/formula/ruby.rb [hunk 1]
```diff
@@ -8,7 +8,7 @@ module Formula
   # The formula may only consist of numbers and the symbols and functions
   # listed in SYMBOLS and FUNCTIONS
   class Ruby < NestFormula
-    SYMBOLS = %w{+ - ( ) [ ] . * , / || &&}.freeze
+    SYMBOLS = %w[+ - ( ) \[ \] . * , / || && { } ].freeze
```
Comment (ethn) [inline mod/calculated_metrics/lib/formula/ruby.rb:11]: Do `{` and `}` have the same meaning in Wolfram?

**Review by ethn — COMMENTED** [review]
I want to be sure we're not reintroducing things that work in ruby but not Wolfram; that causes problems when people start with a working ruby formula and then add a wolfram function to it.

File: mod/calculated_metrics/lib/formula/ruby.rb (hunk 1 above)
Comment (xithan) [reply mod/calculated_metrics/lib/formula/ruby.rb:11]: No, but now they have in Ruby the same meaning they have in Wolfram.

**Comment by xithan** [comment]
We are introducing things that work in Wolfram but not in Ruby (see the specs  https://github.com/wikirate/wikirate/pull/934/files#diff-60fc292aa68bbfc8b8a1053254344584R73)

**Review by ethn — APPROVED** [review]

**Pull request merged**